app/static/js/sidebar.js

```diff
@@ -11,6 +11,7 @@ var options = {
   ]
 };
 var map = L.map("map", options);
+map.fitBounds(L.latLngBounds(L.latLng(15, 17), L.latLng(2.4, 0.85)))
```

Comment:
@christian-rli - I did this because the initial view is too much zoomed in for small screens. However if I zoom in, I can't zoom back out due to zoom restrictions. Do you have an idea on how to set this ?

Reply:
options.minZoom and options.maxZoom are set when entering national level and state level. So if you'd like to change those for the national level, change those values in 
`function adapt_view_to_national_level()`
or initially.